### ISSUE TITLE
81-test_cmp_cli.t has errors related to lsof on multiple platforms.

### ISSUE THREAD

**Issue by slontis** [openssl/openssl#12514]
It no longer hangs.. just fails now.

On aix_ppc - I think requiring the app to run as root in order to run lsof is probably not a good thing.
The error is
```
ok 1 - CMP app CLI basic options
# 
CMP info: received from mock server PKIStatus: accepted 
# CMP warning: server will not be able to handle signature-protected requests since -srv_trusted is not given
# CMP info: will contact http://mock server:1/ via API
# CMP info: sending CR
# CMP info: received CP
# CMP info: received 'waiting' PKIStatus, starting to poll for response
# CMP info: sending POLLREQ
# CMP info: received CP
# CMP info: received ip/cp/kup after polling
# CMP info: sending CERTCONF
# CMP info: received PKICONF
# CMP info: received 1 enrolled certificate(s), saving to file 'test.cert.pem'
../../../../build-aix7_ppc64/util/wrap.pl ../../../../build-aix7_ppc64/apps/openssl cmp -config "" -use_mock_srv -srv_ref 'mock server' -srv_cert server.crt -srv_secret 'pass:test' -poll_count 1 -rsp_cert signer_only.crt -cmd cr -subject /CN=any -newkey signer.key -recipient /O=openssl_cmp -secret 'pass:test' -ref 'client under test' -certout test.cert.pem => 0
ok 2 - CMP app with -use_mock_srv and -poll_count 1
lsof: can't open /dev/mem: Permission denied
lsof: can't open /dev/kmem: Permission denied
Could not load program ../../../../build-aix7_ppc64/apps/openssl:
	Dependent module /usr/lib/libssl.a(libssl64.so.3) could not be loaded.
	Member libssl64.so.3 is not found in archive 
lsof: can't open /dev/mem: Permission denied
lsof: can't open /dev/kmem: Permission denied
Cannot start or find the started CMP mock server at ../test/recipes/81-test_cmp_cli.t line 222.
# Looks like you planned 7 tests but ran 2.
# Looks like your test exited with 1 just after 2.
Dubious, test returned 1 (wstat 256, 0x100)
Failed 5/7 subtests 
```

On solaris11_sparc64_sunC515 it has the following error
Does this one need to set up a path??
```
# CMP info: received 1 enrolled certificate(s), saving to file 'test.cert.pem'
../../../../build-solaris11_sparc64_sunc515/util/wrap.pl ../../../../build-solaris11_sparc64_sunc515/apps/openssl cmp -config "" -use_mock_srv -srv_ref 'mock server' -srv_cert server.crt -srv_secret 'pass:test' -poll_count 1 -rsp_cert signer_only.crt -cmd cr -subject /CN=any -newkey signer.key -recipient /O=openssl_cmp -secret 'pass:test' -ref 'client under test' -certout test.cert.pem => 0
ok 2 - CMP app with -use_mock_srv and -poll_count 1
Can't exec "lsof": No such file or directory at ../test/recipes/81-test_cmp_cli.t line 297.
Use of uninitialized value in pattern match (m//) at ../test/recipes/81-test_cmp_cli.t line 297.
cmp: Error starting accept
CMP info: using OpenSSL configuration file 'server.cnf'
CMP error: <NULL> : calling bind()
CMP error: unable to bind socket
Can't exec "lsof": No such file or directory at ../test/recipes/81-test_cmp_cli.t line 297.
Use of uninitialized value in pattern match (m//) at ../test/recipes/81-test_cmp_cli.t line 297.
Cannot start or find the started CMP mock server at ../test/recipes/81-test_cmp_cli.t line 222.
# Looks like you planned 7 tests but ran 2.
# Looks like your test exited with 2 just after 2.
Dubious, test returned 2 (wstat 512, 0x200)
Failed 5/7 subtests
```

**Comment by slontis** [comment]
@DDvO  travis master also has an issue also.
See https://travis-ci.com/github/openssl/openssl/jobs/364129064

**Comment by DDvO** [comment]
I've just submitted PR #12537 with a workaround for hopefully all issues of this sort.

**Comment by DDvO** [comment]
@slontis, since I just merged #12537 this issue should be gone - can you confirm?

**Comment by slontis** [comment]
Thanks will let you know once the CI loop runs.

**Comment by slontis** [comment]
> @slontis, since I just merged #12537 this issue should be gone - can you confirm?

This appears to address the issue on multiple platforms.. Thanks for your efforts.
